source/common/signal/signal_action.h

```diff
@@ -18,10 +18,10 @@ namespace Envoy {
  *
  * These signals are handled:
  *   SIGABRT
```

Comment:
Why isn't this one quoted? It should be consistent with the rest of them, even if the spell checker doesn't require it.

Reply:
Done.